packages/benefit-price/__tests__/index.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
Awesome Joel. Could you add a few tests for the different props and their results?

Reply:
Thanks Joel!